src/validation.cpp

```diff
@@ -4046,13 +4046,14 @@ bool CChainState::RollforwardBlock(const CBlockIndex* pindex, CCoinsViewCache& i
     return true;
 }
 
-bool CChainState::ReplayBlocks(const CChainParams& params, CCoinsView* view)
+bool CChainState::ReplayBlocks(const CChainParams& params)
 {
     LOCK(cs_main);
 
-    CCoinsViewCache cache(view);
+    CCoinsView& db = this->CoinsDB();
```

Comment:
70e969495191a89070a1308393a5749b37b11e8d

`this->` 👀

Reply:
Hah yeah, I guess we have yet to formalize anything around this but I prefer prefixing method calls with `this->` for clarity. See [the related IRC discussion](http://www.erisian.com.au/bitcoin-core-dev/log-2019-05-06.html#l-572).

Reply:
re: https://github.com/bitcoin/bitcoin/pull/16743#discussion_r319456959

At some point I tried to allow `lowerCase()` for object methods as way to distinguish calls that use the current object from calls that don't (#14635), but agree that `this->` helps without that option.